packages/node/src/indexer/dictionary/v2/utils.ts

```diff
@@ -0,0 +1,50 @@
+// Copyright 2020-2024 SubQuery Pte Ltd authors & contributors
+// SPDX-License-Identifier: GPL-3.0
+
+import { Formatter } from '@ethersproject/providers';
+import { IBlock } from '@subql/node-core';
+import { DictionaryQueryCondition } from '@subql/types-core';
+import { EthereumBlock } from '@subql/types-ethereum';
+import {
+  formatBlockUtil,
+  formatLog,
+  formatTransaction,
+} from '../../../ethereum/utils.ethereum';
+import {
+  EthDictionaryLogConditions,
+  EthDictionaryTxConditions,
+  RawEthBlock,
+} from './types';
+
+export function rawBlockToEthBlock(block: RawEthBlock): IBlock<EthereumBlock> {
+  try {
+    const formatter = new Formatter();
```

Comment:
I don't think we need to worry about it for now because we don't have receipts. But we should apply the formatter changes from here https://github.com/subquery/subql-ethereum/blob/main/packages/node/src/ethereum/ethers/op/op-provider.ts#L19 to this formatter/ have a global formatter